packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -27,12 +27,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
     /**
      * @notice The current Semver of the FaultDisputeGame implementation.
      */
-    string internal constant VERSION = "0.0.1";
+    string internal constant VERSION = "0.0.2";
 
     /**
      * @notice The max depth of the game.
      */
-    uint256 internal constant MAX_GAME_DEPTH = 63;
+    uint256 internal constant MAX_GAME_DEPTH = 4;
```

Comment:
Why do we change this?

Reply:
Testing purposes; Tree padding isn't implemented yet 😄

Reply:
Sweet, thanks 👍